resources/scripts/build_standard_library.py

```diff
@@ -208,17 +282,31 @@ def clone_module(module_link):
 def get_required_module(module_name, module_list):
     module_names = [module[FIELD_NAME] for module in module_list]
     if module_name in module_names:
-        print_info("Building up to the module: " + module_name)
         return module_name
     else:
         print_error("Module not found in the list: " + module_name)
 
 
+def get_version(module):
+    global version_dict
+    with open(GRADLE_PROPERTIES) as properties:
+        for line in properties:
+            if re.match("^version=", line):
+                module_name = module[FIELD_NAME].split("-")[-1]
+                if module_name == "jballerina.java.arrays":
+                    module_name = "JavaArrays"
+                elif module_name == "oauth2":
+                    module_name = "OAuth2"
+                else:
+                    module_name = module_name.capitalize()
+                version_dict[module_name] = line.split("=")[1].strip()
```

Comment:
We can use `version_key` to update the dictionary. (The version key of the constant module is  `stdlibConstraintVersion`)
```python
version_dict[module['version_key']] = module['module_version']

```

Reply:
Fixed with 33d788f226ccf744b0fe702ece2da53c1259f4f7